src/modules/main/navbar/user-section/user-details/UserDetails.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
```

Comment:
Forward ref error in console when using this menu:
![image](https://user-images.githubusercontent.com/22937810/97800342-36a07400-1c3d-11eb-8911-2dcd27861910.png)